android/src/main/kotlin/com/isvisoft/flutter_screen_recording/FlutterScreenRecordingPlugin.kt

```diff
@@ -36,7 +37,7 @@ class FlutterScreenRecordingPlugin(
     var mMediaProjectionCallback: MediaProjectionCallback? = null
     var mVirtualDisplay: VirtualDisplay? = null
     var mDisplayWidth: Int = 1280
-    var mDisplayHeight: Int = 800
```

Comment:
Doesn't have meaning as it's going to be calculated by the mDisplayWidth later on.